Makefrag

```diff
@@ -28,6 +32,10 @@ COMMA := ,
 SBT ?= java -Xmx$(JVM_MEMORY) -Xss8M -XX:MaxPermSize=256M -jar $(base_dir)/sbt-launch.jar
 SHELL := /bin/bash
 
+ifdef STABILIZE
+CHISEL_OPTIONS := -n $(long_name) --with-aspect freechips.rocketchip.linting.rule.StabilizeNamesAspect --lint ambiguous-module-names
```

Comment:
why does this use with-aspect?

Reply:
I'm using the aspect to emit `ModuleNameAnnotation`s so I don't need to modify the chisel code for modules like `Queue` and `TLMonitor`.